helpers/multi.js

```diff
@@ -0,0 +1,8 @@
+module.exports.symbol = Symbol("Multi Brand Symbol");
+
+module.exports.multi = (...values) => {
+    return {
+        " __multiBrand": module.exports.symbol,
+        ...values,
+    };
+};
```

Comment:
`__multiBrand` is just an implementation detail, we might switch to `unique type` once typescript would add it, so I think just that should be enough:

```ts
module.exports.multi = (...values) => values;
```